src/svg-to-vectordrawable.js

```diff
@@ -648,7 +660,10 @@ JS2XML.prototype.refactorData = function(data, floatPrecision, fillBlack, tint)
         elemPaths.forEach(elem => {
             // Fill
             if (elem.hasAttr('fill')) {
-                if (!/^url\(#.*\)$/.test(elem.attr('fill').value) && elem.attr('fill').value !== 'none') {
+                if (elem.attr('fill').value === 'none') {
```

Comment:
Here we remove any `fill="none"` attributes since the vector drawable default behaviour is to not fill.